examples/social/README.md

```diff
@@ -0,0 +1,173 @@
+Metagraph - Data API - Social
+=============================
+
+This example demonstrates a basic social media use case using the Data API. In the example, a client can send four types of signed data updates to a metagraph: one to create a post, another to edit a post, another to delete a post, and another to subscribe to a user. These updates are validated before being merged into the snapshot state.
+
+Here's how the social media system works:
+
+-   Each user can create, edit, and delete their own posts.
+-   Each user can subscribe to other users to follow their posts.
+-   Posts and subscriptions are validated to ensure their integrity and consistency.
+-   Users cannot create duplicate posts with the same content.
+
```

Comment:
One of the main goals of this example is to show an external data store being used to store calculated state. We should describe in detail how that's implemented here.

Reply:
Updated